yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -441,37 +441,25 @@ revision 2019-01-02 {
     </figure>
   </section>
 </section>
-<section title="Import by derived revision" anchor="import">
-  <t><xref target="RFC7950"/> and <xref target="RFC6020"/> allow YANG module "import" statements to optionally require the imported module to have a particular
-  revision date.  In practice, importing a module with an exact revision date is often too restrictive because it
-  requires the importing module to be updated whenever any change to the imported module occurs.  The alternative choice
-  of using an import statement without any revision date statement is also not ideal because the importing module may
-  not work with all possible revisions of the imported module.</t>
-
-  <t>Instead, it is desirable for an importing module to specify a "minimum required revision" of a module that it is
-  compatible with, based on the assumption that later revisions derived from that "minimum required revision" are also
-  likely to be compatible.  Many possible changes to a YANG module do not break importing modules, even if the changes
-  themselves are not strictly backwards-compatible.  E.g., fixing an incorrect pattern statement or description for a
-  leaf would not break an import, changing the name of a leaf could break an import but frequently would not, but
-  removing a container would break imports if that container is augmented by another module.</t>
-
-  <t>The ietf-revisions module defines the "revision-or-derived" extension statement, a substatement to the YANG "import"
-  statement, to allow for a "minimum required revision" to be specified during import:
+<section title="TBD" anchor="import-overview">
+  <t><xref target="RFC7950"/> and <xref target="RFC6020"/> allow YANG module "import" statements to optionally require the imported module to have a specific revision date.  In practice, importing a module with an exact revision date can be too restrictive because it requires the importing module to be updated whenever any change to the imported module occurs, and hence section <xref target="guidelines_authors"/> suggests that authors do not restrict YANG module imports to exact revision dates.</t>
+
+  <t>Instead, for conformance purposes (section 5.6 of <xref target="RFC7950"/>), the recommended approach for defining the relationship between specific YANG modules revisions is to specify the relationships outside of the YANG modules, e.g., via YANG library <xref target="RFC8525"/>, YANG packages <xref target="I-D.ietf-netmod-yang-packages"/>, a filesystem directory containing a set of consistent YANG module revisions, or a git repository label.</t>
```

Comment:
What about "relationship between specific YANG module revisions" and change the explicit mention of git to "revision control system"?

Reply:
I've fixed the first one.  What about "revision control system commit label" for the second?